extension/extension.go

```diff
@@ -76,7 +77,11 @@ func (updateRequest *UpdateRequest) FilterForUpdates(allExtensionsMap *map[strin
 	for _, extensionBeingChecked := range *updateRequest {
 		foundExtension, ok := (*allExtensionsMap)[extensionBeingChecked.ID]
 		if ok {
-			if !foundExtension.Blacklisted && CompareVersions(extensionBeingChecked.Version, foundExtension.Version) < 0 {
+			status := CompareVersions(extensionBeingChecked.Version, foundExtension.Version)
+			if !foundExtension.Blacklisted && status <= 0 {
+				if status == 0 {
```

Comment:
Should this maybe also be done when status > 0?
What does Chrome do in the case you have a newer extension?

Reply:
When status is > 0, component number on the client has a version number higher than the component available on the server. I created a custom request for the google component updater to check the response and the server returns `noupdate`.

But I think returning error in this case is fine, since a lower component version on the client side is unexpected. This might be a result of malformed/malicious/corrupted files in the user directory.

Reply:
I think we should match it with noupdate, but we don't have to do that in the context of this task.